### PR TITLE
Testing cc

### LINKED ISSUE #32: test DailyReading component
- [x] test that a card is being rendered
- [x] test that each element of the card is present
- [x] test that a card is labeled as 'reversed' when it has a `reversed` value of `true`

### LINKED ISSUE #33: Create sad path for landing page.

### PR DESCRIPTION
**Pull request by CalebCyphers**
## Description

This pr adds and refines tests on `App.js`, as well as the `DailyReading.js` component, and adds `Proptypes` to `DailyReading`.
## Types of changes

- [x] Add tests to test files

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Issues

- Closes #32
- Closes #33

### CODE REVIEW

**Review by kathrynljackson — COMMENTED** [review]
All tests are passing!!!

File: src/App/App.test.js [hunk 1]
```diff
@@ -6,33 +6,43 @@ import { fetchCards } from '../fetch/fetch-requests';
 jest.mock('../fetch/fetch-requests');
 
 describe('App', () => {
+
```
Comment (chadrick-d-dev) [inline src/App/App.test.js:9]: Great additions! Go Team!

File: src/DailyReading/DailyReading.test.js [hunk 2]
```diff
@@ -3,51 +3,60 @@ import { render, screen, waitForgit } from '@testing-library/react';
...
+
+  let mockCardReversed = {
+    "reversed": true,
+    "image": "https://i.imgur.com/QBKZRaT.jpg",
+    "title": "The Bard",
+    }
+
+    beforeEach(() => {
+      render(
+        <DailyReading card={mockCard} />
```
Comment (chadrick-d-dev) [inline src/DailyReading/DailyReading.test.js:25]: This is lovely! Makes perfect sense to me!

**Review by chadrick-d-dev — COMMENTED** [review]
Great contributions!

**Pull request merged**